src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/getlantern/go-loggly"
 	"github.com/getlantern/golog"
-	"github.com/stretchr/testify/assert"
+	"github.com/getlantern/testify/assert"
```

Comment:
Weird, GitHub was showing an outdated diff.  I took care of these a while back.  If you checkout obfs4nodep and grep for "getlantern/testify" in src/github.com/getlantern, you won't find anything.